libs/utils/utils.js

```diff
@@ -109,6 +109,12 @@ export const [setConfig, getConfig] = (() => {
       config.codeRoot = conf.codeRoot ? `${origin}${conf.codeRoot}` : origin;
       config.locale = getLocale(conf.locales);
       document.documentElement.setAttribute('lang', config.locale.ietf);
+      try {
```

Comment:
Instead of a try-catch, I would default to `ltr` if config, locale, ietf, textInfo or direction is undefined.

Reply:
@jckautzmann thx for the input. This is the default behavior according to [this page](https://www.w3.org/International/questions/qa-html-dir#rtlsetup) if not set. That's how it worked before this changes.

I don't think we should **explicitly** store/set the default value if we don't really know what the actual direction of the content is, could still be rtl.

There is actually a third option `auto`, to let the agent figure out the direction, but according to docs thats only to be used to set the `dir` attribute on forms and text inputs when entering content at run-time.